dashboard/src/shared/ResourceRef.ts

```diff
@@ -21,6 +21,14 @@ export function fromCRD(
   return ref;
 }
 
+// TODO: (minelson) Update to use API resourceRef type once old model removed.
```

Comment:
Nitpick: I'd use the same comment format to ease the search term when we want to find for them.